wpeverest/everest-forms
# Dev - amoCRM Module

### All Submissions:






### Changes proposed in this Pull Request:



This PR add amoCRM new module in Everest Forms.

### How to test the changes in this Pull Request:
Note : Redirect URL : https://yoursite.com/wp-admin/?evf_amocrm_auth=true

1. Create amoCRM id to get Integration id and Secret key. [amoCRM](https://www.kommo.com/)
2. Enable amoCRM module from dashboard.
3. Goto EVF Global Settings > Integrations > amoCRM.
4. Enter integration id and  click on Get Access Code.
5. Login to amoCRM and choose account. You will get Access code and Referer url.
6. Copy them and paste it.
7. Connect amoCRM and Goto EVF Form Settings > Integration > amoCRM.
8. Map fields and preview form and submit.
9. Check whether data are sync or not.

Dependency PR [PRO PR](https://github.com/wpeverest/everest-forms-pro/pull/952)

### Types of changes:




### Other information:




### Changelog entry

> Dev - amoCRM Module
